packages/gatsby-theme-docs/src/components/config-context.ts

```diff
@@ -25,6 +26,7 @@ const ConfigContext = createContext<Config>({
   selfLearningFeatures: [],
   aiAssistantApiBaseUrl: '',
   aiAssistantTopbarButton: false,
+  enableCanonicalUrls: false,
```

Comment:
Because you defined the default here.

Reply:
For some reason this default value (if set to true, as it should be) doesn't work... so I had to resort to defaulting to false and then use the logic above to actually use `true` as default. Strange but I couldn't figure out the reason